docs/migration.md

```diff
@@ -0,0 +1,195 @@
+# Prometheus 2.0 Migration Guide.
+
+In line with our [compatibility promise](TODO), the Prometheus 2.0 release contains
+a number of backwards incompatible changes.  This document aims to offer guidance on
+migrating from Prometheus 1.8 to Prometheus 2.0.
+
+## Flags
+
+The format of the Prometheus command line flags have changed.  Instead of a
+single dash, all flags now use a double dash. Common flags (`--config.file`,
+`--web.listen-address` and `--web.external-url`) are still the same but beyond
+that, almost all the storage-related flags have been removed.
+
+Some notable flags which have been removed:
+- `-alertmanager.url` In Prometheus 2.0, the command line flags for configuring
+  a static Alertmanager URL have been removed.  Alertmanager must now be
+  discovered via service discovery, see [Alertmanager service discovery](#amsd).
+
+- `-log.format` In Prometheus 2.0 logs can only be streams to standard out.
+
+- `-query.staleness-delta` Prometheus 2.0 introduces a new mechanism for
+  handling staleness, see [Staleness](#staleness).
+
+- `-storage.local.*` Prometheus 2.0 introduces a new storage engine, as such all
+  flags relating to the old engine have been removed.  For information on the
+  new engine, see [Storage](#storage).
+
+- `-storage.remote.*` Prometheus 2.0 has removed the deprecated remote storage
+  flags.  To write to InfluxDB, Graphite, or OpenTSDB use the relevant storage
```

Comment:
These flags were already not working

Reply:
Yeah, instead, link to https://prometheus.io/docs/prometheus/2.0/storage/#remote-storage-integrations for remote storage in Prometheus 2.0.